pages/docs/reference/coding-conventions.md

```diff
@@ -726,6 +727,201 @@ foo {
 
 </div>
 
+### Trailing commas
+
+A trailing comma is a comma sign after the last item of the element series:
+
+<div class="sample" markdown="1" theme="idea" mode="kotlin" data-highlight-only>
+
+```kotlin
+class Person(
+    val name: String,
+    val surname: String,
+    val age: Int, // trailing comma
+)
+```
+
+</div>
+
+Using trailing commas has several benefits:
+
+* It makes version-control diffs cleaner: all focus is on changed value.
+* It's easy to add or reorder elements: no need to add or delete the comma sign if you manipulate elements.
+* It simplifies code generation (for example, for object initializers). The last element can also have the comma sign.
+
+Using trailing commas are optional: your code will work without them. But Kotlin style guide encourages the use of trailing commas at the declaration site and leaves it optional for the call site.
+
+To enable trailing commas for the IntelliJ formatter, go to __Settings | Editor | Code Style | Kotlin__, 
+open the __Other__ tab and enable the __Use trailing comma__ option.
+
+Kotlin supports trailing commas in the following cases:
```

Comment:
Not all supported cases are listed here. There is also support trailing comma in type arguments, type parameters and destructuring declaration.

Reply:
Actually, this list of supported cases seems like reference-style documentation. I personally don't think that Coding Conventions page is a place for describing the language syntax in this way. 
I'd write in in more general with words with several (3-5) **typical** examples.

Reply:
OK, I just thought that here describes all of the supported cases (due to the phrase `Kotlin supports trailing commas in the following cases`). BTW it can be merged in the following way: `value and type arguments`, `value and type parameters`.

Reply:
I've added `Type arguments and type parameters` and `Destructuring declarations` instead of merging